src/logic/asset-converter.ts

```diff
@@ -22,7 +26,14 @@ export default function createAssetConverter(commandTrigger: ICommandLine): IAss
 
   async function convert(fileToConvert: string, options: CrunchConversionOptions): Promise<ConversionResult> {
     const args = getArguments(fileToConvert, options)
-    return await commandTrigger.execute('crunch', args)
+
+    const metricLabels = { size: Number(options.size) }
+    const metricTimer = metrics.startTimer('conversion_duration_seconds', metricLabels)
+
+    return await commandTrigger
+      .execute('crunch', args)
+      .then((result) => result)
+      .finally(() => metricTimer.end(metricLabels))
```

Comment:
Please use idiomatic constructs inside async functions
```suggestion
  try {
    return await commandTrigger
      .execute('crunch', args)
  } finally {
     metricTimer.end(metricLabels)
  }
```